### DISCUSSION TITLE
Eastern boundary upwelling system SST

### DISCUSSION THREAD

**Discussion by justinsmall** [NCAR/omwg_dev#50] [Biases]
Attaching a plot of SST bias in EBUS from G-case g.e23_a16g.GJRAv4.TL319_t232_zstar_N65.2024.004. This case is forced by JRA55-do. Annual mean, years 280-309. 

The biases are OK, relative to standard-res POP 
![MOM6-zSST-bias png](https://github.com/NCAR/omwg_dev/assets/22841083/ba836490-e088-4e31-9106-1f16d0822791)

**Comment by justinsmall** [comment]
@gustavo-marques   I am also attaching here a figure of SST bias in 3 older cases: High-res POP forced by CORE (CORE-HR), high-res POP forced by JRA55-do (JRA55-HR), standard-res POP forced by JRA55-do (JRA55-LR). The MOM6 z* case is somewhat better than standard-res POP forced by JRA55-do.
[Fig2.pdf](https://github.com/NCAR/omwg_dev/files/15392501/Fig2.pdf)

**Comment by justinsmall** [comment]
A possible reason for the improved EBUS in MOM6, relative to POP, is that the horizontal grid in MOM6 (2/3) is closer to that of JRA55 (~1/2) than POP (~1), so winds are not degraded so much by interpolation.

**Comment by justinsmall** [comment]
@gustavo-marques  I am also attaching results from new coupled runs. This is from the z* case b.e23_alpha17f.BLT1850.ne30_t232.091  The results are very typical for coupled runs with a ~ 1deg. atmosphere (ne30), see e.g. https://doi.org/10.1038/s43247-023-00681-0
I would say, the biases are not especially large for this resolution.
![CESM3-z-BLT1850 ne30_t232 091SST-bias 000006](https://github.com/NCAR/omwg_dev/assets/22841083/f37c1df9-1d90-47a0-b135-1eae6807ec37)

**Comment by justinsmall** [comment]
Here is the corresponding G-case 
![MOM6-hybridSST-bias png](https://github.com/NCAR/omwg_dev/assets/22841083/31bda0ce-b7a6-4dc1-b74a-bfb4fa69a81f)
plot from hybrid coordinate forced by JRA55-do. g.e23_a16g.GJRAv4.TL319_t232_hycom1_N75.2024.005. Biases are very similar to z*.

**Comment by justinsmall** [comment]
@gustavo-marques This is the coupled case with hybrid coordinates: b.e23_alpha17f.BLT1850.ne30_t232.092
Compared to the coupled z* case, it is globally warmer, and so biases in EBUS appear larger. But if you look at the zonal gradient of SST bias near the coast, it is similar to z*.
Again, these seem "normal" for this resolution, no particular surprises.
![CESM3-hybrid-BLT1850 ne30_t232 092SST-bias 000006](https://github.com/NCAR/omwg_dev/assets/22841083/9d8fd1c6-3e5d-4174-89fe-1a864bf8b34d)

**Comment by gustavo-marques** [comment]
@justinsmall: thank you for these plots, they are great!
